kano_world/connection.py

```diff
@@ -43,7 +41,7 @@ def _remove_sensitive_data(request_debug):
 def request_wrapper(method, endpoint, data=None, headers=None,
                     session=None, files=None, params=None):
     if method not in ['put', 'get', 'post', 'delete']:
-        return False, 'Wrong method name!'
```

Comment:
I've had this changed on my VM for so long and not bothered to commit it - good change!